### PR TITLE
Maintenance bits

### PR DESCRIPTION
**Pull request by c410-f3r**
Each commit should be read individually

### CODE REVIEW

File: pallets/liquidity-provider/src/lib.rs [hunk 1]
```diff
@@ -56,26 +57,30 @@ decl_module! {
         #[weight = 0]
         pub fn attest(
```
Comment (olanod) [inline pallets/liquidity-provider/src/lib.rs:58]: What happens when we attest `USDv`, seems possible but it shouldn't.

File: pallets/liquidity-provider/src/lib.rs [hunk 2]
```diff
@@ -56,26 +57,30 @@ decl_module! {
...
+            asset: Asset,
             balance: Balance<T>
         ) -> dispatch::DispatchResult
         {
             let who = ensure_signed(origin)?;
+            do_attest::<T>(who, asset, balance)
+        }
 
+        #[weight = 0]
+        pub fn mint(
```
Comment (olanod) [inline pallets/liquidity-provider/src/lib.rs:69]: Still not sure about exposing this as a public dispatchable call, maybe as a private function of the module?

Comment (c410-f3r) [reply pallets/liquidity-provider/src/lib.rs]: mint method was removed alongside with mint members

File: pallets/liquidity-provider/src/lib.rs [hunk 3]
```diff
@@ -56,27 +55,21 @@ decl_module! {
...
+            match asset {
+                Asset::Usdv => return Err(crate::Error::<T>::MustNotBeUsdv.into()),
+                Asset::Collateral(collateral) => {
+                    let who = ensure_signed(origin)?;
+                    do_attest::<T>(who.clone(), Asset::Usdv, balance)?;
+                    T::Collateral::deposit(collateral.into(), &who, balance)?;
+                    T::Collateral::reserve(collateral.into(), &who, balance)?;
+                    Self::deposit_event(RawEvent::Attestation(who, collateral.into()));
+                    Ok(())
+                }
```
Comment (c410-f3r) [inline pallets/liquidity-provider/src/lib.rs:71]: Since every attestation is also a mint, everything that is not a stable collateral will fail

**Pull request merged**